File: estrutura de dados/1 - Entrada de Dados/questoes/questao06.c
#include <stdio.h>

// Questão 6: Cálculo de Imposto sobre Salário (10%)
void questao6() {
    float salario, imposto;
    
    printf("Digite o salario: ");
    scanf("%f", &salario);
    
    imposto = salario * 0.10;
    
    printf("Salario: %.4f\n", salario);
    printf("Imposto: %.4f\n", imposto);
}

int main() {
    questao6();
    
    return 0;
}

File: orientado a objetos/00-entrada-de-dados/exercicios-entrada-de-dados/src/Questao06.java
import java.util.Scanner;

public class Questao06 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double salario, imposto, salario_final;

        System.out.print("Digite o salário: ");
        salario = sc.nextDouble();

        imposto = salario * 0.1;
        salario_final = salario - imposto;

        System.out.printf("Salário: R$%.2f\n", salario);
        System.out.printf("Imposto: R$%.2f\n", imposto);
        System.out.printf("Salário final: R$%.2f\n", salario_final);
    }
}
